pkg/resources/ray/horizontalpodautoscaler.go

```diff
@@ -0,0 +1,57 @@
+package ray
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	autoscalingv2beta2 "k8s.io/api/autoscaling/v2beta2"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+
+	dcv1alpha1 "github.com/dominodatalab/distributed-compute-operator/api/v1alpha1"
+)
+
+const (
+	scaleTargetKind            = "Deployment"
+	stabilizationWindowSeconds = 60
```

Comment:
This window time feels like something that might end up being configurable because of cluster constraints

Reply:
that's easy to make configureable. I was just starting with what I don't know but that seems like a reasonable field to expose and we can use a default value of 60 when not specified.